requirements.txt

```diff
@@ -1,9 +1,7 @@
-aiohttp>=3.8.6
 cachetools==5.3.3
 google-auth==2.22.0
 pyasn1==0.6.0
 pyasn1-modules==0.4.0
 rsa==4.9
-requests>=2.6.0
```

Comment:
`google.auth.transport.requests` [requires `requests` to be installed](https://github.com/googleapis/google-auth-library-python/blob/c3ea09fd8b9ee8f094263fdc809c59d9dfaa3124/google/auth/transport/requests.py#L27) - given it forms the backbone of authenticating requests for this library, IMO it would not be prudent to have dual support or try and monkeypatch.